layers/meta-balena-allwinner/conf/machine/orangepi-plus2.conf

```diff
@@ -6,4 +6,5 @@ require conf/machine/include/sun8i.inc
 
 KERNEL_DEVICETREE = "sun8i-h3-orangepi-plus.dtb"
 UBOOT_MACHINE = "orangepi_plus_defconfig"
-PREFERRED_VERSION_linux-mainline = "4.17.3"
+PREFERRED_PROVIDER_virtual/kernel = "linux"
+PREFERRED_VERSION_linux = "4.19.76%"
```

Comment:
I think we could remove this as it will anyway default to 4.19.76 right?

Reply:
Yes, it will default to 4.19.76. Will remove the PREFERRED_VERSION.